src/org/labkey/snd/SNDManager.java

```diff
@@ -129,7 +131,7 @@ public static SNDManager get()
 
     public static UserSchema getSndUserSchema(Container c, User u)
```

Comment:
This seems dangerous?  This is used a lot and does no permission checking and there are over 60 usages.  This should take an explicit parameter or have a name that clearly indicates what is going on.

Reply:
Ok I created a different function (getSndUserSchemaAdminRole) which includes admin role and using only where needed

Reply:
For most usages would  contextualRole = ReaderRole be sufficient?